rules/internal/bincapz.yara

```diff
@@ -3,9 +3,14 @@ rule bincapz : harmless {
 		__bincapz__ = "true"
 		description = "match the bincapz binary, to omit it unless --ignore-self=false"
     strings:
-        $report = "bincapz/pkg/report"
-		$behavior = "bincapz.Behavior"
-		$render = "bincapz/pkg/render"
+		$b_behavior = "bincapz.Behavior"
+		$f_action = "bincapz/pkg/action"
+		$f_bincapz = "bincapz/pkg/bincapz"
+		$f_compile = "bincapz/pkg/compile"
+		$f_profile = "bincapz/pkg/profile"
+		$f_render = "bincapz/pkg/render"
+        $f_report = "bincapz/pkg/report"
+        $f_version = "bincapz/pkg/version"
     condition:
-        2 of them
+        $b_behavior and all of ($f_*)
```

Comment:
Requiring every f* will make this fragile when testing against older and newer bincapz releases. I'd change it back or switch it to 3 of them.

Reply:
Updated in [`f670fbc` (#323)](https://github.com/chainguard-dev/bincapz/pull/323/commits/f670fbcc1c263d38e06f5b02cb9f44bf15e9d0b7).